caesar_rest/index_route.py

```diff
@@ -11,4 +11,24 @@
 
 @index_bp.route('/')
 def index():
-	return render_template('index.html')
+	if oidc.user_loggedin:
+		return render_template('index.html', username=oidc.user_getfield('email'))
+        	#return 'Welcome {email}'.format(email=oidc.user_getfield('email'))
+    	else:
+        	return 'Not logged in, <a href="/login">Log in</a> '
+
+
+@index_bp.route('/login')
+@oidc.require_login
+def login():
+    user_info = oidc.user_getinfo(['preferred_username', 'email', 'sub', 'name'])
```

Comment:
This is not used at the moment, as we redirect to "/" and do nothing else with user data.